src/promptflow/promptflow/exceptions.py

```diff
@@ -7,6 +7,8 @@
 from enum import Enum
 from functools import cached_property
 
+from azure.core.exceptions import HttpResponseError
```

Comment:
PR title optimizing exceptions is too broad.
May say:
[Error Handling] reduce unknow rate for error category and target

Reply:
good catch, done.